elements/pl-file-editor/pl-file-editor.js

```diff
@@ -141,27 +149,51 @@ window.PLFileEditor.prototype.initSettingsButton = function (uuid) {
           })
         );
       }
+
+      var fontSizeList = ["12px", "14px", "16px", "18px", "20px", "22px", "24px"];
+      var fontSelect = that.modal.find('#modal-' + uuid + '-fontsize');
+      fontSelect.empty();
+      for (const entries in fontSizeList) {
+        fontSelect.append(
+          $('<option>', {
+            value: fontSizeList[entries],
+            text: fontSizeList[entries],
+            selected: localStorage.getItem('pl-file-editor-fontsize') === fontSizeList[entries],
+          })
+        );
+      }
     });
     that.modal.modal('show');
     sessionStorage.setItem('pl-file-editor-theme-current', that.editor.getTheme());
+    sessionStorage.setItem('pl-file-editor-fontsize-current', that.editor.getFontSize());
     that.modal.find('#modal-' + uuid + '-themes').change(function () {
       var theme = $(this).val();
       that.editor.setTheme(theme);
     });
+    that.modal.find('#modal-' + uuid + '-fontsize').change(function () {
+      var fontSize = $(this).val();
+      that.editor.setFontSize(fontSize);
+    });
   });
 
   this.saveSettingsButton.click(function () {
     var theme = that.modal.find('#modal-' + uuid + '-themes').val();
+    var fontsize = that.editor.getFontSize();
```

Comment:
I believe should use the value from the `<select>` element, just like `theme` does on the previous line.

Reply:
This was resolved in db966ae.